gprimitives/src/lib.rs

```diff
@@ -162,15 +162,58 @@ impl FromStr for ActorId {
                 .map_err(|_| ConversionError::InvalidHexString)?;
         } else {
             let raw_address: [u8; 32] = RawSs58Address::from_ss58check(s)
-                .map(Into::into)
-                .map_err(|_| ConversionError::InvalidSs58Address)?;
+                .map_err(|_| ConversionError::InvalidSs58Address)?
+                .into();
             actor_id.0[..].copy_from_slice(&raw_address);
         }
 
         Ok(actor_id)
     }
 }
 
+#[cfg(feature = "serde")]
```

Comment:
pls impl these for MessageId and CodeId as well
(or as usual open an issue for faster merging of this)

Reply:
should we also provide serde for H256, H160...?

Reply:
Wasn't it done by default?

Reply:
in origin crate